Libs/MRML/Core/vtkMRMLSegmentationStorageNode.cxx

```diff
@@ -138,6 +140,7 @@ void vtkMRMLSegmentationStorageNode::InitializeSupportedReadFileTypes()
   this->SupportedReadFileTypes->InsertNextValue("Segmentation (.seg.vtm)");
   this->SupportedReadFileTypes->InsertNextValue("Segmentation (.nrrd)");
   this->SupportedReadFileTypes->InsertNextValue("Segmentation (.vtm)");
+  this->SupportedReadFileTypes->InsertNextValue("Segmentation (.nii.gz)");
```

Comment:
Add .hdr (nifti format with separate header and pixel data) and .nii (uncompressed single-file nifti).